pkg/scheduler/backend/queue/active_queue.go

```diff
@@ -61,14 +61,46 @@ type activeQueuer interface {
 // underLock() method should be used to protect these methods.
 type unlockedActiveQueuer interface {
 	unlockedActiveQueueReader
-	AddOrUpdate(pInfo *framework.QueuedPodInfo)
+	add(pInfo *framework.QueuedPodInfo, event string)
```

Comment:
nit: let's clarify what "event" is for

```suggestion

	// add adds a new pod to the activeQ.
	// event should show which event to trigger this addition, which will be used for the metric recording. 
	add(pInfo *framework.QueuedPodInfo, event string)
```